code/game/machinery/firealarm.dm

```diff
@@ -97,6 +98,19 @@ FIRE ALARM
 		alarm(rand(30/severity, 60/severity))
 	..()
 
+/obj/machinery/firealarm/examine(mob/user)
+	. = ..()
+	switch(buildstage)
+		if(FIRE_ALARM_FRAME)
+			. += "<span class='notice'>It is <i>bolted</i> to the wall and the <b>circuit</b> slot is empty.</span>"
+		if(FIRE_ALARM_UNWIRED)
+			. += "<span class='notice'>The circuit inside could be <i>pried out</i> or could be <b>wired</b>.</span>"
+		if(FIRE_ALARM_READY)
+			if(wiresexposed)
+				. += "<span class='notice'>Its wires would need to be <i>cut</i> to access the circuit or panel could <b>screwed</b> closed.</span>"
```

Comment:
```suggestion
				. += "<span class='notice'>Its wires would need to be <i>cut</i> to access the circuit, otherwise the panel could <b>screwed</b> closed.</span>"
```